tensorboard/plugins/profile/overview_page/overview-page.html

```diff
@@ -97,9 +97,9 @@
        text-decoration: underline;
      }
     </style>
-    <template is="dom-if" if="[[_show_error_message]]" restamp=true>
-      <div class="errorMessage"><span>[[_error_message]]</span></div></template>
-    <template is="dom-if" if="[[!_show_error_message]]" restamp=true>
+    <div class="errorMessage" hidden=[[!_show_error_message]]>
```

Comment:
what is the technical real behind `dom-if` -> `hidden`?

Reply:
Using dom-if, when the tool detached and re-attached, causing data races that the dom in one of the path is not initialized fast enough. This causes the line chart not showing up. Now we will always initialize the dom in both path, to prevent this bug.